tls/s2n_signature_scheme.c

```diff
@@ -138,20 +151,23 @@ const struct s2n_signature_scheme s2n_rsa_pss_rsae_sha256 = {
         .iana_value = TLS_SIGNATURE_SCHEME_RSA_PSS_RSAE_SHA256,
         .hash_alg = S2N_HASH_SHA256,
         .sig_alg = S2N_SIGNATURE_RSA_PSS_RSAE,
+        .libcrypto_nid = NID_rsassaPss,
```

Comment:
Is this the right nid? Why does this id not include the hash algorithm?

Reply:
I'm actually not sure. I am still looking into why the id does not include the hash.

Reply:
Resolved offline; I will make a github issue to track this.